extensions/amp-story-interactive/0.1/amp-story-interactive-img.css

```diff
@@ -65,20 +65,27 @@
   background-size: cover !important;
 }
 
+.i-amphtml-story-interactive-img-option-img::before {
+  background-color: var(--i-amphtml-interactive-option-post-color) !important;
```

Comment:
There is a flaky diff that might be due to this being set before there is data.
Moving this line to the post state might fix the flaky diff.